theano/gpuarray/cudnn_defs.py

```diff
@@ -123,6 +253,8 @@ class CuDNNV6(CuDNNV51):
                                                 ('CUDNN_CONVOLUTION_BWD_FILTER_ALGO_FFT_TILING', 'fft_tiling'),
                                                 ctype='cudnnConvolutionBwdFilterAlgo_t')
 
+    deterministic_bwd_filter_algorithms = CuDNNV51.deterministic_bwd_filter_algorithms + ('fft_tiling',)
```

Comment:
You didn't reimplement that for cudnn v7. Can you confirm this isn't a forget and it don't have more deterministic algo?

Reply:
Yes I confirm (checked now), no new deterministic algorithms.